From 59c3c511626e5f76202353f1220fe7aaf8db521d Mon Sep 17 00:00:00 2001
From: TwinShip <sake19@naver.com>
Date: Fri, 29 Jul 2022 19:19:39 +0900
Subject: [PATCH] add "core technique"

---
 ICT Express/Smart_Farm.tex | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/ICT Express/Smart_Farm.tex b/ICT Express/Smart_Farm.tex
index d473b44..ff3c6fe 100644
--- a/ICT Express/Smart_Farm.tex	
+++ b/ICT Express/Smart_Farm.tex	
@@ -40,5 +40,5 @@ IoT 기반 플랫폼은 데이터 분석을 통해 도출된 결과를 기반으
 하고, 관개시스템을 구역마다 다르게 제어하는 시스템을 구축한다.
 
 \section{System Architecture}
-
+우리는 머신러닝 기반 의사결정 시스템과 Closed loop 기반 소프트웨어 업데이트 알고리즘으로 핵심 연구 내용을 구분한다.
 \end{document}
-- 
2.33.1.windows.1

